src/rating/rating.ts

```diff
@@ -85,11 +96,30 @@ export class NgbRating implements OnInit {
     this.hover.emit(value);
   }
 
-  ngOnInit(): void {
-    this._oldRate = this.rate;
-    this.range = this._buildTemplateObjects();
+  /**
+   * internal
+   */
+  getFillValue(index: number): number {
```

Comment:
Remove the JS doc comment. The way it is written right now it would show up in the API doc.